### DISCUSSION TITLE
In routing, how are ways handled that match multiple priority statements in routing.xml?

### DISCUSSION THREAD

**Discussion by uhkeller** [osmandapp/OsmAnd#17855] [Q&A]
Sorry if I missed it, but I can't find any explanation on how the following situation is handled. Suppose there is a way tagged with `highway=secondary` and `sidewalk=yes`, and in the active profile in `routing.xml` there are these two statements matching it:

```xml
<select value="1.2" t="sidewalk" v="yes"/>
<select value="0.9" t="highway" v="secondary"/>
```

What priority is assigned to this way? The first (1.2)? The last (0.9)? The average? Both somehow?

Thanks in advance for your help!

**Comment by Sonwon1** [comment]
Couldn't you just create a test route to see how the routing algorithm handles it?

**Comment by uhkeller** [comment]
I could and have started doing it, but figuring out exactly what's happening in this way seems pretty difficult. Also, it seems like a rather obvious question that should maybe be addressed in the documentation part of `routing.xml`, don't you think?
